scripts/performance-data-migration/encounterMigration.sql

```diff
@@ -19,21 +20,16 @@ BEGIN
     SELECT encounter_type_id, uuid
     FROM input.encounter_type;
 
-  UPDATE input.encounter_type
-    SET creator = admin_id
-    WHERE creator IS NOT NULL;
-
-  UPDATE input.encounter_type
-    SET retired_by = admin_id
-    WHERE retired_by IS NOT NULL;
-
-  call debugMsg(1, 'input updated');
-
-  INSERT INTO encounter_type (name, description, creator, date_created, retired, retired_by,
+  INSERT INTO encounter_type (`name`, description, creator, date_created, retired, retired_by,
                               date_retired, retire_reason, uuid, view_privilege, edit_privilege,
                               changed_by, date_changed)
-    SELECT name, description, creator, date_created, retired, retired_by, date_retired,
-      retire_reason, uuid, view_privilege, edit_privilege, changed_by, date_changed
+    SELECT `name`, description,
+      CASE WHEN creator IS NULL THEN NULL ELSE admin_id END AS creator,
+      date_created, retired,
+      CASE WHEN retired_by IS NULL THEN NULL ELSE admin_id END AS retired_by,
+      date_retired, retire_reason, uuid, view_privilege, edit_privilege,
+      CASE WHEN changed_by IS NULL THEN NULL ELSE admin_id END AS changed_by,
```

Comment:
You have done it because this is faster?

Reply:
I changed it because it is much safer for users of a script if the script does not change source database.